services/graphite/parser.go

```diff
@@ -117,7 +117,8 @@ func (p *Parser) Parse(line string) (models.Point, error) {
 	}
 
 	if math.IsNaN(v) || math.IsInf(v, 0) {
-		return nil, fmt.Errorf(`field "%s" value: '%v" is unsupported`, fields[0], v)
+		// Graphite ignores NaN values with no error
+		return nil, nil
```

Comment:
This should return a specific error type and be handled (and ignored) higher up.  Returning a `nil` error here is problematic with the standard go `if err != nil {}` idiom.  You now have to check both `err` and `point` not being nil which is not what a caller would typically expect or do.  This func should either return a `Point` or an `error`.